### PR TITLE
*: add Catalan translation

### PR DESCRIPTION
**Pull request by Seifer23**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ca/linux/apt-file.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# apt-file
```
Comment (navarroaxel) [inline pages.ca/linux/apt-file.md:1]: The example `--regexp` is missing, you translated from the Spanish page, but the Spanish page is outdated. We based all the translations in the main «English» pages. You can check the translation status (and outdated) here: https://lukwebsforge.github.io/tldri18n/
Could you please sync this translation with the English version?

File: pages.ca/linux/apt-get.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# apt-get
```
Comment (navarroaxel) [inline pages.ca/linux/apt-get.md:1]: `apt-get autoclean` example is missing here too.

Comment (Seifer23) [reply pages.ca/linux/apt-get.md:1]: solved

File: pages.ca/linux/a2disconf.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> Desactiva un fitxer de configuració d'Apache en sistemes operatius basats en Debian.
+> Més Informació: <https://manpages.debian.org/latest/apache2/a2disconf.8.en.html>.
```
Comment (navarroaxel) [inline pages.ca/linux/a2disconf.md:4]: Why the `i` in mayus? we don't use `Title Case`
```suggestion
> Més informació: <https://manpages.debian.org/latest/apache2/a2disconf.8.en.html>.
```

File: pages.ca/linux/systemctl.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`systemctl {{start|stop|restart|reload}} {{unitat}}`
+
+- Mostra l'estat d'una unitat:
+
+`systemctl status {{unitat}}`
+
+- Habilita/Deshabilita una unitat perquè s'inicii en l'arrencada:
+
+`systemctl {{enable/disable}} {{unitat}}`
```
Comment (navarroaxel) [inline pages.ca/linux/systemctl.md:24]:
```suggestion
`systemctl {{enable|disable}} {{unitat}}`
```

File: pages.ca/linux/dos2unix.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+> Reemplaça CRLF amb CR.
+> Més informació: <https://manned.org/dos2unix>.
+
+- Canvia els salts de línia en un arxiu:
+
+`dos2unix {{nom_del_arxiu}}`
+
+- Crea una còpia amb salts de línia en format Unix:
+
+`dos2unix -n {{nom_del_arxiu}} {{nom_del_arxiu}}`
```
Comment (navarroaxel) [inline pages.ca/linux/dos2unix.md:13]:
```suggestion
`dos2unix -n {{nom_del_arxiu}} {{nou_nom_del_arxiu}}`
```

File: pages.ca/linux/a2dismod.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# a2dismod
+
+> Desactiva un mòdul Apache en sistemes operatius basats en Debian.
+> Més Informació: <https://manpages.debian.org/latest/apache2/a2dismod.8.en.html>.
```
Comment (navarroaxel) [inline pages.ca/linux/a2dismod.md:4]:
```suggestion
> Més informació: <https://manpages.debian.org/latest/apache2/a2dismod.8.en.html>.
```

File: pages.ca/linux/a2dissite.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# a2dissite
+
+> Desactiva un host virtual d'Apache en sistemes operatius basats en Debian.
+> Més Informació: <https://manpages.debian.org/latest/apache2/a2dissite.8.en.html>.
```
Comment (navarroaxel) [inline pages.ca/linux/a2dissite.md:4]:
```suggestion
> Més informació: <https://manpages.debian.org/latest/apache2/a2dissite.8.en.html>.
```

File: pages.ca/linux/a2enconf.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# a2enconf
+
+> Activa un fitxer de configuració d'Apache en sistemes operatius basats en debian.
+> Més Informació: <https://manpages.debian.org/latest/apache2/a2enconf.8.en.html>.
```
Comment (navarroaxel) [inline pages.ca/linux/a2enconf.md:4]:
```suggestion
> Més informació: <https://manpages.debian.org/latest/apache2/a2enconf.8.en.html>.
```

File: pages.ca/linux/a2enmod.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+# a2enmod
+
+> Activa un mòdul d'Apache en sistemes operatius basats en Debian.
+> Més Informació: <https://manpages.debian.org/latest/apache2/a2enmod.8.en.html>.
```
Comment (navarroaxel) [inline pages.ca/linux/a2enmod.md:4]:
```suggestion
> Més informació: <https://manpages.debian.org/latest/apache2/a2enmod.8.en.html>.
```

File: pages.ca/linux/a2query.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# a2query
+
+> Recupera la configuració del temps d'execució d'Apache en sistemes operatius basats en Debian.
+> Més Informació: <https://manpages.debian.org/latest/apache2/a2query.html>.
```
Comment (navarroaxel) [inline pages.ca/linux/a2query.md:4]:
```suggestion
> Més informació: <https://manpages.debian.org/latest/apache2/a2query.html>.
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

Great job!

**Pull request merged**